.github/workflows/collection-continuous-integration.yml

```diff
@@ -0,0 +1,158 @@
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.runner-python-version }}
+      - name: Set up pip cache
+        uses: actions/cache@v1
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ github.ref }}-sanity-containers
+          restore-keys: |
+            ${{ runner.os }}-pip-
+            ${{ runner.os }}-
+      - name: Install Ansible ${{ matrix.ansible-version }}
+        run: >-
+          python -m pip install --user ${{ matrix.ansible-version }}
+      - name: Download migrated collection artifacts
+        uses: actions/download-artifact@v1
+        with:
+          name: collection
+          path: .cache/collection-tarballs
+      - name: Install the collection tarball
+        run: >-
+          ~/.local/bin/ansible-galaxy collection install .cache/collection-tarballs/*.tar.gz
+      - name: Run collection sanity tests
+        run: >-
+          ~/.local/bin/ansible-test sanity --color --requirements --docker --python "${{ matrix.python-version }}" -vvv
+        working-directory: >-
+          /home/runner/.ansible/collections/ansible_collections/containers/podman
+
+  unit-test-collection-via-containers:
+    name: Units in container ${{ matrix.container-image }}
+    needs:
+      - build-collection-artifact
+    runs-on: ${{ matrix.runner-os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        runner-os:
+          - ubuntu-latest
+        runner-python-version:
+          - 3.7
+        ansible-version:
+          - git+https://github.com/ansible/ansible.git@devel
+        container-image:
+          - fedora31
+          #- ubuntu1804
+          - centos8
+          #- opensuse15
+          #- fedora30
+          - centos7
+          #- opensuse15py2
+          #- ubuntu1604
+          #- centos6
+    steps:
+      - name: Set up Python ${{ matrix.runner-python-version }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.runner-python-version }}
+      - name: Set up pip cache
+        uses: actions/cache@v1
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ github.ref }}-units-containers
+          restore-keys: |
+            ${{ runner.os }}-pip-
+            ${{ runner.os }}-
+      - name: Install Ansible ${{ matrix.ansible-version }}
+        run: >-
+          python -m pip install --user ${{ matrix.ansible-version }}
+      - name: Download migrated collection artifacts
+        uses: actions/download-artifact@v1
+        with:
+          name: collection
+          path: .cache/collection-tarballs
+      - name: Install the collection tarball
+        run: >-
+          ~/.local/bin/ansible-galaxy collection install .cache/collection-tarballs/*.tar.gz
+      - name: Run collection unit tests
+        run: |
+          [[ ! -d 'tests/unit' ]] && echo This collection does not have unit tests. Skipping... || \
```

Comment:
This conditional is unnecessary. It was only needed in `migrate.py` because sometimes we had resulting collections with no tests. If you know that you have tests, remove this or it may start silently start skipping testing at some point in the future.

Reply:
Yeah, but actually we don't have unittests right now. But very want to have them. :) So maybe we'll leave it until we have tests and then will remove this condition.

Reply:
In this case, I'm pretty sure you could just add something like `if: false` to the step definition. This way, it'd visually show up as skipped, not succeeded in the CI.

Reply:
Moreover, you could add another step that will always fail if unit tests exist, so it'd prompt you to remove `if: false` once somebody adds actual tests.